src/main/java/com/wolox/training/models/User.java

```diff
@@ -76,6 +59,27 @@ public void setBooks(List<Book> books) {
     this.books = books;
   }
 
+  @Id
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "USER_SEQ")
+  @SequenceGenerator(name = "USER_SEQ", sequenceName = "USER_SEQ")
+  private Long id;
+
+  @Column(nullable = false, unique = true)
+  private String userName;
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false)
+  private LocalDate birthDate;
+
+  @ManyToMany(cascade = {CascadeType.MERGE, CascadeType.REFRESH})
+  private List<Book> books;
```

Comment:
Just as a convention, remember that in Java we declare first the attributes and then the methods :)